backend/drm/drm.c

```diff
@@ -50,6 +50,13 @@ bool check_drm_features(struct wlr_drm_backend *drm) {
 		return false;
 	}
 
+	if(drmGetCap(drm->fd, DRM_CAP_CRTC_IN_VBLANK_EVENT, &cap) ||
+	   !(cap & DRM_CAP_CRTC_IN_VBLANK_EVENT)) {
```

Comment:
Style: tabs, no spaces :P

Reply:
*sigh* I still need to create a proper c-style for emacs. Fixed.